src/bindings/pygaia/scripts/classification/classification_project_template.yaml

```diff
@@ -26,6 +26,19 @@ filelist: %(filelist)s
 groundtruth: %(groundtruth)s
 
 
+# Seed used to randomly assign tracks to folds. Use 'None' to use clock value. This will 
+# produce slightly different results on different trials)
+seed: %(seed)s
+
+# Cluster mode controls how subprocesses are called. Set it to True to explicitly use the subprocess
+# module to open a new python process and call each classificationTask as a script. Otherwise, this
+# call will be performed as a function.
+clusterMode: %(clusterMode)s
+
+# List of parameters that have to be excluded before creating applying the prepocessings
+excludedDescriptors: [ 'metadata.tags*' ]
```

Comment:
If the input file does have this descriptor then the file fails to load into the dataset. The json_to_sig script removes this descriptor. We should make it so that this isn't a failure if it doesn't exist